mtgjson4/compile_mtg.py

```diff
@@ -200,7 +200,7 @@ def transpose_tokens(
 
 
 def add_tcgplayer_fields(
-    group_id: int, cards: List[Dict[str, Any]]
+    group_id: int, set_code: str, cards: List[Dict[str, Any]]
```

Comment:
Function docs must be updated

Reply:
Updated. Triple-checked all function docs.